sematic/config/config.py

```diff
@@ -185,9 +210,13 @@ def switch_env(env: str) -> None:
         )
 
     set_config(EnvironmentConfigurations[env].value)
-    logger = logging.getLogger(__name__)
     logger.info("Switch to env {} whose config is {}".format(env, get_config()))
 
+    # TODO #302: implement sustainable way to upgrade sqlite3 DBs
+    # we are equivalating cloud env with postgresql, but this not might be true
+    if env != EnvironmentConfigurations.cloud:
```

Comment:
Might it be better to directly check whether the DB URL is for sqlite?

Reply:
Due to unfathomable laziness. Done.